apps/web/src/pages/SearchRequests/SearchPage/components/SearchPools.tsx

```diff
@@ -58,29 +79,66 @@ const SearchPools = ({
           },
         )}
       </p>
+      <p data-h2-margin="base(x.5, 0, x1, 0)">
+        <ExternalLink
+          mode="inline"
+          type="button"
+          color="secondary"
+          href={paths.pool(pool.id || "")}
+          newTab
+        >
+          {intl.formatMessage({
+            defaultMessage: "View the job poster for this recruitment process",
+            id: "2Ljgvn",
+            description:
+              "Link message that shows the job poster for the recruitment process.",
+          })}
+        </ExternalLink>
+      </p>
+      <p data-h2-margin="base(x1, 0, 0, 0)">
+        {intl.formatMessage({
+          defaultMessage:
+            "These essential skills were assessed during the process:",
+          id: "VN6uCI",
+          description:
+            "Text showing the essentials skills assessed during the process",
+        })}
+      </p>
+      <p>{skillsArray}</p>
       <p data-h2-margin="base(x1, 0, 0, 0)">
         {intl.formatMessage(
           {
-            defaultMessage: "Pool Owner: {name}",
-            id: "o+a0IN",
-            description: "Text showing the owner of the HR pool.",
+            defaultMessage: "Process run by {team} at {departments}",
+            id: "6yDdnk",
+            description: "Text showing the team and department of the HR pool.",
           },
           {
-            name: pool?.owner
-              ? getFullNameHtml(
-                  pool?.owner.firstName,
-                  pool?.owner.lastName,
-                  intl,
-                )
+            team: pool?.team
+              ? getLocalizedName(pool.team.displayName, intl)
               : intl.formatMessage({
-                  defaultMessage: "N/A",
-                  id: "AauSuA",
-                  description: "Not available message.",
+                  defaultMessage: "Digital Community Management Team",
+                  id: "gfO156",
+                  description: "Text showing the team of the HR pool.",
+                }),
+            departments: departmentsArray
+              ? insertBetween(
```

Comment:
used insertBetween as the leading and trailing spaces were trimmed for "and" without that

Reply:
@Jerryescandon Are you ok with adding comma as separator instead of "and" ?

Reply:
Jerbo agreed with that. Done!